ertviz/models/realization.py

```diff
@@ -1,4 +1,47 @@
+import pandas as pd
+import math
+
+
 class Realization:
     def __init__(self, realization_schema):
-        self.name = realization_schema["name"]
-        self.data = realization_schema["data"]
+        self._name = realization_schema["name"]
+        self._data = realization_schema["data"]
+        self._univariate_misfits_df = self._extract_univariate_misfits(
+            realization_schema.get("univariate_misfits")
+        )
+        self._summarized_missfits_value = self._extract_summary_misfits(
+            realization_schema.get("summarized_misfits")
+        )
+
+    def _extract_univariate_misfits(self, schema):
+        if bool(schema):  # this account for not None and empty dict
```

Comment:
Often i find it more readable doing the return early and near the check of the variable 
```
if not bool(schema):
    return None

misfits_ = list(schema.values())
..
..
.
return df
```

Reply:
will do!